content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -0,0 +1,82 @@
+---
+title: Channel Parameters
+section: realtime
+index: 1
+jump_to:
+  Help with:
+    - Overview#overview
+    - Supported channel parameters#supported-parameters
+    - Using channel parameters#using-parameters
+    - Using channel parameters with non-Ably transports#using-parameters-with-other-transports
+    - Next steps#next-steps
+---
+
+h2(#overview). Overview
```

Comment:
We need to state which SDKs support this API or not, and update https://www.ably.io/download/sdk-feature-support-matrix.

Reply:
I think a statement on the pages themselves as mentioned in your last comment will suffice.